Magick++/fuzz/encoder_fuzzer.cc

```diff
@@ -33,8 +33,7 @@ static ssize_t EncoderInitializer(const uint8_t *Data, const size_t Size, Magick
   return 0;
 }
 
-extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
-{
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
```

Comment:
Looks like this got into your PR by accident?

Reply:
Indeed.